test/built-ins/Math/atan2/S15.8.2.5_A16.js

```diff
@@ -20,8 +20,7 @@ var ynum = 3;
 for (var i = 0; i < ynum; i++)
 {
   assert.sameValue(
-    Math.atan2(y[i], x),
-    -0,
+    Math.atan2(y[i], x), -0,
```

Comment:
This should definitely have the -0 on a separate line; iow, it was correct before. jsbeautify bug?